recipes-soletta/soletta/soletta_git.bb

```diff
@@ -4,7 +4,7 @@
 
 DESCRIPTION = "Soletta library and modules"
 SECTION = "examples"
-DEPENDS = "glib-2.0 libpcre pkgconfig python3-jsonschema-native icu curl libmicrohttpd"
+DEPENDS = "glib-2.0 libpcre pkgconfig python3-jsonschema-native icu curl libmicrohttpd mosquitto"
```

Comment:
not mosquitto, you should depend on libmosquitto1, no? or is this the recipe name x package-name?

just check that none of the systemd-service, mosquitto (server) are being in the resulting image.